app/views/users/show.html.erb

```diff
@@ -58,7 +58,12 @@ end %>
         </td>
       </tr>
     </table>
-    <div id="pmb-data" data-pmb-api="<%= configatron.pmb_api %>" data-pmb-template="<%= @label_template_id || 0 %>"></div>
+    <div
+      id="pmb-data"
+      data-pmb-api="<%= configatron.pmb_api %>"
+      data-swipecard_pmb_template="<%= configatron.swipecard_pmb_template %>"
```

Comment:
Could you remove the setting of @label_template_id from the controller. It should remove the need for an API call to PMB.

Reply:
Also, we're typically using kebab case for data-attributes (I think the vue-style-guide enforces this, although this isn't a vue component). You might find you'd need to switch to camel case for data retrieval (you do when going via dataset) but I'm not sure how jquery wraps this interface.